rudder-agent/SOURCES/patches/cfengine/07-define-classes-when-package-is-up-to-date.patch

```diff
@@ -1,13 +0,0 @@
-diff -ur a/cf-agent/verify_packages.c b/cf-agent/verify_packages.c
```

Comment:
This is a duplicate patch with 0017-define-classes-when-package-is-up-to-date.patch. They do the same thing, therefore the second doesn't apply, causing errors.